samples/tcp-ingress/worker.js

```diff
@@ -0,0 +1,11 @@
+
+export default {
+  async fetch(req) {
+    return new Response("ok");
+  },
+
+  connect({inbound, cf}) {
```

Comment:
Reviewer note:

`inbound` is a `ReadableStream`

The expected return value is `ReadableStream` (or more correctly, `Promise<ReadableStream>`)